toolbar.js

```diff
@@ -20,6 +22,11 @@ const listingToolbar = {
     customToolbar.appendChild(userCount.createElement());
     customToolbar.appendChild(titleFilter.createElement());
     issueTable.parentNode.insertBefore(customToolbar, issueTable);
+
+    // Add header for merge request.
+    issueTable
+      .querySelector("thead tr")
+      .appendChild(mergeRequestStatus.createElement());
```

Comment:
Since this functionality really doesn't relate to the custom toolbar could you add it directly to `drupal-content-queue.js` and encapsulate everything 1 function like

`mergeRequestStatus.addColumn()`